pkg/image/apiserver/registry/imagestreamimport/rest_test.go

```diff
@@ -110,8 +113,8 @@ func TestImportSuccessful(t *testing.T) {
 			},
 			expectedTagEvent: imageapi.TagEvent{
 				Created:              now,
-				DockerImageReference: "registry.com/namespace/image:mytag",
-				Image:                "image",
```

Comment:
This seems bad (that we had tag here before)

Reply:
What code led to this outcome before?  If this could happen in the wild it’s a huge bug.  Was it just that the fake image had what we would consider an invalid digest?